src/pages/Home/components/Portfolio/NetworkWidget/Assetlist.tsx

```diff
@@ -98,7 +98,7 @@ export function Assetlist({ assetList }: AssetListProps) {
             justifyContent="space-between"
             margin="0 -16px"
             padding="4px 16px"
-            key={token.symbol}
+            key={token.type === TokenType.NATIVE ? token.symbol : token.address}
```

Comment:
With (at least) two separate BTC.b addresses, the entries were getting duplicated.
Using a proper (unique) `key` fixed the problem on my side.